src/fsharp/vs/ServiceDeclarations.fsi

```diff
@@ -118,13 +118,15 @@ type internal FSharpDeclarationListItem =
     member IsOwnMember : bool
     member MinorPriority : int
     member FullName : string
+    member NamespaceToOpen : string option
 
 type internal CompletionItem =
     { Item: Item
       Kind: CompletionItemKind
       IsOwnMember: bool
       MinorPriority: int
-      Type: TyconRef option }
+      Type: TyconRef option 
```

Comment:
Again I'm growing concerned about the extensive leakage of internal compiler data structures out into FSHarp.Editor - here ``TyconRef`` stands out.  It's not new in this PR though.